src/messaging/SecureChannelMgr.cpp

```diff
@@ -0,0 +1,269 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
```

Comment:
Please remove copyright line (years are annoying to update).

@woody-apple  - is this ok with you? I find updating years tedious.

Reply:
I prefer a separate PR to update the Copyright header part unanimously after we finalize what should be kept and what should be removed.